optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/drools/testgen/reproducer/TestGenCorruptedScoreReproducer.java

```diff
@@ -66,10 +66,10 @@ public boolean isReproducible(TestGenKieSessionJournal journal) {
         } catch (RuntimeException e) {
             if (e.getMessage() != null && e.getMessage().startsWith("No fact handle for ")) {
                 // this is common when removing insert of a fact that is later updated - not interesting
-                logger.debug("    Can't remove insert: {}", e.toString());
+                logger.debug("    Can't remove insert: {}", (Object) e);
```

Comment:
@ge0ffrey I'll undo this together with some other work on testgen.

I was intentionally using e.toString() as I wanted to get both the message and FQCN. For the sake of avoiding the warning about implicit toString() calls I could rework it to `e.getClass().getName() + ": " + e.getMessage()` but that's not nice.

Another solution could be to fix SLF4J to support this use case...